### PR TITLE
fix/node env session parse

### PR DESCRIPTION
**Pull request by marcosnils**
- fixes issue parsing DAGGER_SESSION_PORT and DAGGER_SESSION_TOKEN

### CODE REVIEW

**Review by sipsma — APPROVED** [review]

**Comment by slumbering** [comment]
Hola @marcosnils 👋. Quick question, why the `yarn.lock` did change ?

File: sdk/nodejs/connect.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ export async function connect(
       )
     }
     client = new Client({
-      host: `http://127.0.0.1:${daggerSessionPort}/query`,
+      host: `127.0.0.1:${daggerSessionPort}`,
```
Comment (dolanor) [inline sdk/nodejs/connect.ts:71]: @marcosnils : can you add some documentation to the Client for this new format (no scheme prefix, no path suffix)?

**Comment by marcosnils** [comment]
> Hola @marcosnils wave. Quick question, why the `yarn.lock` did change ?

Because I did a `yarn install` in my workspace and yarn bumped automatically all the PATCH versions given how dependencies are defined. I'd assume this is very common?

File: sdk/nodejs/connect.ts (hunk 1 above)
Comment (marcosnils) [reply sdk/nodejs/connect.ts:71]: There's no new format @dolanor! this was a bug that when using the `DAGGER_SESSION_PORT` and `DAGGER_SESSION_TOKEN` variables the URL was constructed incorrectly. There's no need for documentation here.

**Comment by slumbering** [comment]
> > Hola @marcosnils wave. Quick question, why the `yarn.lock` did change ?
> 
> Because I did a `yarn install` in my workspace and yarn bumped automatically all the PATCH versions given how dependencies are defined. I'd assume this is very common?

Ok make sense, not a big deal.

**Comment by aluzzardi** [comment]
@marcosnils Do we need to ship a hotfix release or can this wait until next week for regular releases?

**Pull request merged**